src/App.js

```diff
@@ -1,11 +1,19 @@
+import React from 'react';
 import Calculator from './component/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <Calculator />
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
- [ ] [OPTIONAL] you can remove this part if you aren't using the state so it's useless